src/index.ts

```diff
@@ -0,0 +1,18 @@
+"use strict";
```

Comment:
Have removed this and added as part of the ts build.

Reply:
yep.. I forgot to remove that from every file.